coffee_cmd/src/coffee/mod.rs

```diff
@@ -180,19 +180,27 @@ impl PluginManager for CoffeeManager {
     }
 
     async fn list(&mut self, remotes: bool) -> Result<Value, CoffeeError> {
-        let mut plugin_names: Vec<Plugin> = Vec::new();
+        let installed_plugins_vec: Vec<Plugin> = self.config.plugins.clone();
+        let mut plugin_json = json!({
+            "plugins": serde_json::to_value(&installed_plugins_vec).unwrap(),
+         });
```

Comment:
```suggestion
        let mut plugin_json;
```

Reply:
why this conversation here? we can not do this only if it is necessary? and we avoid a json conversion?